### ISSUE TITLE
Missing documentation/support for "INPUT_OBJECT" metadata examples

### ISSUE THREAD

**Issue by kbailey923** [anvilco/spectaql#84]
Minimal example with a single `createUser` mutation that takes in `CreateUserInput` input type as parameter:

`schema.graphql`
```
type User {
    id: ID!
    firstName: String!
    middleName: String
    lastName: String!
    email: String!
}

input CreateUserInput {
    firstName: String!
    middleName: String
    lastName: String!
    email: String!
}

type CreateUserPayload {
    user: User
}

type Mutation {
    createUser(input: CreateUserInput!): CreateUserPayload!
}
```

`metadata.json`:
```
{
    "INPUT_OBJECT": {
        "CreateUserInput": {
            "inputFields": {
                "firstName": {
                    "documentation": {
                        "example": "John"
                    }
                },
                "middleName": {
                    "documentation": {
                        "example": "Quincy"
                    }
                },
                "lastName": {
                    "documentation": {
                        "example": "Doe"
                    }
                },
                "email": {
                    "documentation": {
                        "example": "johndoe@gmail.com"
                    }
                }
            }
        }
    }
}
```

Expected behavior for generated `CreateUserInput` example:
```
{
    "firstName": "John",
    "middleName": "Quincy",
    "lastName": "Doe",
    "email": "johndoe@gmail.com",
}
```

Actual behavior for generated `CreateUserInput` example:
```
{
    "firstName": "abc123",
    "middleName": "abc123",
    "lastName": "abc123",
    "email": "abc123",
}
```

If metadata examples are currently supported for input objects, could you provide a working example?
Thanks.

**Comment by chemmex** [comment]
I confirm that per field (per input field) metadata for INPUT_OBJECT is not applied to the output. Neither 'example', nor 'undocumented'

**Comment by newhouse** [comment]
Fix coming shortly...

**Comment by newhouse** [comment]
Fixed in `0.8.0`
